Makefile

```diff
@@ -90,6 +90,8 @@ setup:  # the setup steps necessary on developer machines
 test: lint unit cuke  # runs all the tests
 .PHONY: test
 
+test-go: build unit cuke-go lint-go  # runs all tests for Golang
```

Comment:
Given this order do you think we should also change test to lint last?

Reply:
In `test` the cucumber specs are by far the longest, so I want them last there. In Go land, the linter is actually the slowest right now, so it comes last right now.